### PR TITLE
Update to 1.0

### PR DESCRIPTION
**Pull request by mateusortiz**
I did all updates to version 1.0 of Polymer, we must review now. :bowtie:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ If you have [Emmet](https://github.com/emmetio/emmet-atom) installed it will clo
 
 ## Elements
 
-Type the name of [any `core-*` or `paper-*` element](https://www.polymer-project.org/docs/elements/), then hit `tab` to auto complete. Ex:
+Type the name of [any `iron-*` or `paper-*` element](https://elements.www.polymer-project.org/), then hit `tab` to auto complete. Ex:
```
Comment (robdodson) [inline README.md]: change url to `https://elements.polymer-project.org`

File: snippets/paper-menu.cson [hunk 2]
```diff
@@ -0,0 +1,9 @@
+".text.html":
+  "paper menu":
+    "prefix": "paper-menu"
+    "body": """
+    <paper-menu${1: multi}>
+    <paper-item>Item 1</paper-item>
+    <paper-item>Item 2</paper-item>
```
Comment (robdodson) [inline snippets/paper-menu.cson]: indent paper-items

File: snippets/polymer-element-stylesheet.cson [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+        $2
 	    </template>
 	    <script>
 		    Polymer({
-			    $5
+			    is: '$1'
 		    });
 	    </script>
-    </polymer-element>
+    </dom-module>
```
Comment (robdodson) [inline snippets/polymer-element-stylesheet.cson:15]: Use 2 space indent, like other files

**Comment by robdodson** [comment]
Just a few small things to fix

**Comment by mateusortiz** [comment]
sorry Rob for some things, I've fixed all.

**Comment by robdodson** [comment]
Great thanks @mateusortiz !

**Pull request merged**